File: URI/python/1020.py
age_in_days = int(input())

years = age_in_days // 365 
months = (age_in_days % 365) // 30
days = (age_in_days % 365) % 30

print('{} ano(s)'.format(years))
print('{} mes(es)'.format(months))
print('{} dia(s)'.format(days))

File: uri/c/1020.c
#include <stdio.h>

int main() {

    int age;
    scanf("%d", &age);

    printf("%d ano(s)\n", age / 365);
    printf("%d mes(es)\n", age % 365 / 30);
    printf("%d dia(s)\n", age % 365 % 30);

    return 0;

}